dts/bindings/pwm/ite,it8xxx2-pwm.yaml

```diff
@@ -21,14 +21,17 @@ properties:
       type: int
       required: true
       enum:
-            - 0 #PWM_CHANNEL_0
-            - 1 #PWM_CHANNEL_1
-            - 2 #PWM_CHANNEL_2
-            - 3 #PWM_CHANNEL_3
-            - 4 #PWM_CHANNEL_4
-            - 5 #PWM_CHANNEL_5
-            - 6 #PWM_CHANNEL_6
-            - 7 #PWM_CHANNEL_7
+            - 0
+            - 1
+            - 2
+            - 3
+            - 4
+            - 5
+            - 6
+            - 7
+      description: 0 = PWM_CHANNEL_0, 1 = PWM_CHANNEL_1, 2 = PWM_CHANNEL_2,
```

Comment:
use `description: |` for multiline descriptions

Reply:
Ack. Done.